CRM/Core/BAO/Query.php

```diff
@@ -80,4 +101,22 @@ public static function searchAction(&$row, $id) {}
    */
   public static function tableNames(&$tables) {}
 
+  /**
+   * Get the name of the field.
+   *
+   * @param array $values
+   *
+   * @return string
+   */
+  protected static function getFieldName($values) {
+    $name = $values[0];
+    $fieldAliases = self::getLegacySupportedFields();
+    if (isset($fieldAliases[$name])) {
+      CRM_Core_Error::deprecatedFunctionWarning('These parameters should be standardised before we get here');
```

Comment:
@eileen Are you sure this won't be hit by accident?  ie. that `$values` won't already contain standardised fields?

Reply:
@mattwire well as eileen mentioned the search form doesn't seem to hit it nor any current tests so i think its probably sensible to put it in and see if anyone does actually hit it anywhere